.pipelines/ci/ci.yml

```diff
@@ -5,6 +5,7 @@ trigger:
       - master
       - dev/build-features
       - dev/PowerLauncher
```

Comment:
Since we are updating the file, we can remove these two that are not needed anymore.
Also remove them from line 19 and 20.
Thanks.

Reply:
i was going to do that in a different PR :).  these are left overs from Build timeframe

Reply:
@alannt777 feel free to do these and the ones below in the PR section if you want